### ISSUE TITLE
Publish release artifacts to Maven Central

### ISSUE THREAD

**Issue by jvettraino** [di2e/Argo#50]
The Argo releases should be published to Maven central so other products can easily include any of the artifacts as dependencies (things like the JAXB objects, or probe generation code are very handy for other Argo clients to leverage).

**Comment by jmsimpson68** [comment]
Which part of Argo did you have in mind?  I'm doing some refactoring which componentizes the Wireline Format JAXB classes.  It makes it much easier to handle the dependencies.  I wouldn't necessarily push anything that has a .tar.gz release or .war release to Maven Central.  All that being said, I haven't done it before and if you have any advice, I'm all ears.

**Comment by jmsimpson68** [comment]
Jeff, have you gone through the process before?  I've been doing a lot of reading on the subject, but I think I will have a non-trivial learning curve here.

**Comment by jvettraino** [comment]
Hey Jeff,

Yeah I can definitely help you with this.  Let me know if you have specific
questions, or if you just want to walk through them on the phone, that
works as well.

Thanks!

Jeff Vettraino


*Cohesive Integrationsjeff.vettraino@cohesiveintegrations.com
<jeff.vettraino@cohesiveintegrations.com>(602) 332-1377*

**Comment by jmsimpson68** [comment]
Lets do this sometime this week.  I have some other things to do.  I’ll let you know when I have time freeing up.  Hopefully I can give you a day or two heads up.

I’m putting in the command line UI for Argo (which it needs).  That will be it for this release.  Then I’ll do the payload encryption (at least for XML) in the next release.

Turn out the CL UI was more of a pain then I anticipated.  I am co-opting some code I found in Google Code - after reviewing 3 command line UI frameworks (which I really didn’t feel like building myself).

Jeff Simpson
jmsimpson68y@yahoo.com
571-212-0927
